src/vsphere_cpi/Gemfile.lock

```diff
@@ -110,4 +109,4 @@ DEPENDENCIES
   vsphere-automation-vcenter (~> 0.1.0)
 
 BUNDLED WITH
-   2.1.4
+   1.17.2
```

Comment:
All the changes in this file should be reverted. It is not safe to use Bundler `1.17.2` . Please resist from commiting this file. You can keep it as local change by not adding it during `git add -p`

You need to upgrade the bundler on your machine. We can work on this.

Reply:
Please remove this file from the commit by using git rebase.